src/dev-app/cdk-experimental-combobox/panel-content.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Directive, Inject, InjectionToken, Input, OnInit, Optional} from '@angular/core';
+import {AriaHasPopupValue, CdkComboboxPanel} from '@angular/cdk-experimental/combobox';
+
+export const PANEL = new InjectionToken<CdkComboboxPanel>('CdkComboboxPanel');
+
+let id = 0;
+
+@Directive({
+    selector: '[panelContent]',
+    exportAs: 'panelContent',
+    host: {
+        'role': 'role',
+        '[id]': 'dialogId'
+    }
+})
+export class PanelContent<V> implements OnInit {
```

Comment:
Is this just for testing, or do you intend for this to be the ultimate API?

Reply:
So this directive is just for testing/demo purposes. It is the directive that is applied to the custom panel content the user wishes to use, like their own dialog. It handles registering the content with the panel. I did realize it was doing more than I needed it to do in this example so I've toned it down.